kaldb/src/main/java/com/slack/kaldb/chunkManager/ChunkRollOverStrategy.java

```diff
@@ -1,6 +1,14 @@
 package com.slack.kaldb.chunkManager;
 
+import java.io.File;
+
 // TODO: Rename ChunkRollOverPolicyPredicate
 public interface ChunkRollOverStrategy {
   boolean shouldRollOver(long currentBytesIndexed, long currentMessagesIndexed);
+
+  public void setActiveChunkDirectory(File activeChunkDirectory);
+
+  public long getApproximateDirectoryBytes();
```

Comment:
Let's remove this method from the interface. I don't think we need to expose this as a guage in the IndexingChunkManager. If needed, we can expose the same info as a metric in the roll over strategy impl class.